enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -243,6 +243,55 @@ const questionId = 1; // This is the question ID you want to embed
 
 ```
 
+### Embedding a static dashboard
+
+After the SDK is configured, you can embed your dashboard using the `StaticDashboard` component.
+
+The API follows the same configuration as that of public dashboard embeddings, which you can read about (here)[https://www.metabase.com/docs/latest/questions/sharing/public-links#public-embed-parameters]
+
+
+#### Parameters
+
+- **dashboardId**: `DashboardId` (required) – The ID of the dashboard to embed.
+- **parameterQueryParams**: `Query` (required) – Query parameters for the dashboard.
+- **bordered**: `boolean` – Whether the dashboard should have a border.
+- **titled**: `boolean` – Whether the dashboard should display a title.
+- **hideDownloadButton**: `boolean | null` – Whether to hide the download button.
+- **hideParameters**: `string[] | null` – A list of parameters that will not be shown in the set of parameter filters. (More information here)[https://www.metabase.com/docs/latest/questions/sharing/public-links#filter-parameters]
+- **font**: `string | null` – The font to use.
```

Comment:
Just to confirm, if the font prop is `null` or `undefined` would it fallback to the font family defined in the theme config?

Reply:
yep. I'll add that in the comment

Reply:
Let's add this to the README